app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,88 @@
 // Specific CSS for your home-page
+
+
+.container {
+  background-size: cover;
+  background-position: center;
+  padding: 50px 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
A container is a class that should just be setting a width and center it in the screen. Making it a display flex and padding is out of scope and should be applied to another div.
Also your background-size and -position don't do anything as your container doesn't have a background-image

Also there is now a class container-1200 that has been merged in the _setup.scss file so you can use this one and see if it fits your needs.